info/v1/container.go

```diff
@@ -64,10 +64,16 @@ type ContainerSpec struct {
 
 	// Image name used for this container.
 	Image string `json:"image,omitempty"`
+
+	// The ID of the container
+	ContainerId string `json:"container_id,omitempty"`
 }
 
 // Container reference contains enough information to uniquely identify a container
 type ContainerReference struct {
+	// The container id
+	Id string `json:"id"`
```

Comment:
Why is this required for a reference, but optional for the spec? I think these should be consistent, or at least the Reference should have weaker requirements than the spec.